app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -305,9 +306,10 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
 
         startupTelemetryOnCreateCalled(intent.toSafeIntent())
         startupPathProvider.attachOnActivityOnCreate(lifecycle, intent)
-        startupTypeTelemetry = StartupTypeTelemetry(components.startupStateProvider, startupPathProvider).apply {
-            attachOnHomeActivityOnCreate(lifecycle)
-        }
+        startupTypeTelemetry =
+            StartupTypeTelemetry(components.startupStateProvider, startupPathProvider).apply {
+                attachOnHomeActivityOnCreate(lifecycle)
+            }
```

Comment:
Yes, the formatter added the whitespace changes. I left the changes as I thought it would also be a problem for someone else in the future (e.g. having to purposefully avoid formatting)?

Reply:
I think this for the project team to agree on a standard, and if necessary for a project wide formatting PR, rather than mixed in with functional changes.